core/templates/pages/signup-page/modal-templates/license-explanation-modal.template.directive.html

```diff
@@ -3,15 +3,21 @@ <h3 translate="I18N_SIGNUP_WHY_LICENSE"></h3>
 </div>
 
 <div class="modal-body oppia-long-text">
-  <p style="line-height: 1.5;" translate="I18N_SIGNUP_SITE_OBJECTIVE" translate-values="{sitename: siteName}">
+  <p class="oppia-signup-objective-text" translate="I18N_SIGNUP_SITE_OBJECTIVE" translate-values="{sitename: siteName}">
   </p>
 
-  <p style="line-height: 1.5;" translate="I18N_SIGNUP_LICENSE_OBJECTIVE" translate-licenselink="<a href=&quot;https://creativecommons.org/licenses/by-sa/4.0/&quot; target=&quot;_blank&quot;>CC-BY-SA v4.0</a>">
+  <p class="oppia-signup-objective-text" translate="I18N_SIGNUP_LICENSE_OBJECTIVE" translate-licenselink="<a href=&quot;https://creativecommons.org/licenses/by-sa/4.0/&quot; target=&quot;_blank&quot;>CC-BY-SA v4.0</a>">
   </p>
 
-  <p style="line-height: 1.5;" translate="I18N_SIGNUP_WAIVER_OBJECTIVE"></p>
+  <p class="oppia-signup-objective-text" translate="I18N_SIGNUP_WAIVER_OBJECTIVE"></p>
 </div>
 
 <div class="modal-footer">
   <button class="btn btn-secondary" ng-click="cancel()" translate="I18N_SIGNUP_CLOSE_BUTTON"></button>
 </div>
+
+<style>
+  license-explanation .oppia-signup-objective-text {
```

Comment:
I don't see any tag with this name, can you please check whether it's working as expected?

Reply:
Here I found a bug, created an issue for it https://github.com/oppia/oppia/issues/10375. Removed the tag for now.

Reply:
The bug is something different not related to the tag name, right? [Just wanted to confirm!]